index.bs

```diff
@@ -3254,8 +3254,15 @@ credential.
         with more than one [=user account=] at the [=[RP]=].
 
     :   <dfn>displayName</dfn>
-    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display. For example, "Alex Müller" or "田中倫". The
-        [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice more than necessary.
+    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display.
+        [=Clients=] may display this value to help users differentiate between multiple sub-accounts for a
+        given [=user account=] for situations in which every credential uses the same value for
+        {{PublicKeyCredentialEntity/name}}. When such differentiation is not required, this value
+        MAY be set to the same value as {{PublicKeyCredentialEntity/name}}. Examples of suitable
+        values for this identifier include, "Alex Müller" or "田中倫".
+
+        A [=[RP]=] MAY let the user specify this value, and SHOULD NOT restrict the choice
+        more than necessary when doing so.
```

Comment:
Continuing from https://github.com/w3c/webauthn/pull/1932#discussion_r1281092796 and [commit `f09e3d8c` comments](https://github.com/w3c/webauthn/commit/f09e3d8cea148765c5d634b283e9cc81f583ee6e#commitcomment-123354509):

Hm. Relaxing the last sentence from SHOULD to MAY is strictly speaking a normative change, though a backwards compatible one. It probably doesn't matter either way, but I don't think it needs to change either.

I think it reads marginally better with "The [=[RP]=]", if nothing else for consistency with the description of `name` in the previous section. I think "...when doing so" is a fine addition if the requirement is relaxed from SHOULD to MAY.

I'm not sure the idea of "sub-accounts" is necessary, but I wouldn't say it hurts either.

All in all I think this is still fine to merge as is, but since you asked @MasterKale :slightly_smiling_face: here's how I would phrase these changes as a whole:

```suggestion
    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display. For example, "Alex Müller" or "田中倫".
        [=Clients=] may display this value to help users differentiate between multiple [=credentials=] with the same value of {{PublicKeyCredentialEntity/name}}.

        The [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice more than necessary.
        The [=[RP]=] MAY set this to the same value as {{PublicKeyCredentialEntity/name}}.
```

Granted it might look a bit weird to have a SHOULD immediately followed by a conflicting MAY, but I think this should communicate that the latter is fine but the former is preferable when practical.

Reply:
Again, to be clear: I think the PR is fine without the above suggestion, but feel free to use some, all or none of it as you like @MasterKale. :slightly_smiling_face:

Reply:
> ...but I think this should communicate that the latter is fine but the former is preferable when practical.

Ah, I see the subtle difference. Your suggested wording sounds more spec-like, and you captured the ultimate meaning I want to achieve with this effort. I made some changes accordingly.

Reply:
Ok, looks good!